packages/pds/src/repo/prepare.ts

```diff
@@ -26,7 +30,7 @@ import {
 
 // @TODO do this dynamically off of schemas
```

Comment:
ah man we should do this off of blob refs now 😛 

i may take a stab at that soon

Reply:
I was tempted, haha!